host/xtest/regression_4000.c

```diff
@@ -2859,6 +2863,7 @@ static const struct xtest_ac_case xtest_ac_cases[] = {
 			  TEE_MODE_ENCRYPT,
 			  ac_rsaes_oaep_vect10, NULL_ARRAY, WITHOUT_SALT),
 
+#endif
```

Comment:
move before the empty line and recall the condition in an inline comment, i.e.:
`#endif /* !CFG_CRYPTOLIB_NAME_mbedtls || !MBEDTLS_RSA_NO_CRT */`